modules/xfem/doc/content/source/constraints/XFEMSingleVariableConstraint.md

```diff
@@ -1 +1,33 @@
-!template load file=stubs/moose_object.md.template name=XFEMSingleVariableConstraint syntax=/Constraints/XFEMSingleVariableConstraint
\ No newline at end of file
+# XFEMSingleVariableConstraint
+
+!syntax description /Constraints/XFEMSingleVariableConstraint
+
+## Overview
+
+Due to the Phantom Node method's matrix implementation requiring interaction between fragmented
```

Comment:
Without any other intervention, the phantom node method used by MOOSE's XFEM system results in Heaviside enrichment of all solution fields at the locations of interfaces, with zero flux conditions. In many cases, there is a need to impose constraints across the interface on one or more of the solutions fields in a multiphysics XFEM simulation, which is the purpose of XFEMSingleVariableConstraint.

Reply:
Is the first part true? I know the defaults for any variable given a single variable constraint are jump=0 and flux_jump=0, but you still have to specify that the variable has a constraint, right? It may be an arbitrary distinction, but this reads as though if you don't give a variable a constraint, that it will automatically be continuous across an XFEM cut, but is this actually the case?